Index: src/main/java/com/jvschool/entities/DeliveryWayEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/DeliveryWayEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/DeliveryWayEntity.java	(revision )
@@ -7,13 +7,13 @@
  * Created by Людмила on 19.07.2017.
  */
 @Entity
-@Table(name = "DeliveryWay", schema = "myshop_schema")
+@Table(name = "DELIVERY_WAY", schema = "MYSHOP_SCHEMA")
 public class DeliveryWayEntity implements Serializable {
     private int deliveryWayId;
     private String deliveryWayName;
 
     @Id
-    @Column(name = "DeliveryWayId", nullable = false)
+    @Column(name = "DELIVERY_WAY_ID", nullable = false)
     @GeneratedValue(strategy=GenerationType.AUTO)
     public int getDeliveryWayId() {
         return deliveryWayId;
@@ -24,7 +24,7 @@
     }
 
     @Basic
-    @Column(name = "DeliveryWayName", nullable = false, length = 50)
+    @Column(name = "NAME", nullable = false, length = 50)
     public String getDeliveryWayName() {
         return deliveryWayName;
     }
Index: src/main/java/com/jvschool/entities/AddressEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/AddressEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/AddressEntity.java	(revision )
@@ -9,7 +9,7 @@
  * Created by Людмила on 19.07.2017.
  */
 @Entity
-@Table(name = "Address", schema = "myshop_schema")
+@Table(name = "ADDRESS", schema = "MYSHOP_SCHEMA")
 public class AddressEntity implements Serializable {
     private long addressId;
     private String streetAddress;
@@ -24,7 +24,7 @@
 
 
     @Id
-    @Column(name = "AddressId", nullable = false)
+    @Column(name = "ADDRESS_ID", nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public long getAddressId() {
         return addressId;
@@ -35,7 +35,7 @@
     }
 
     @Basic
-    @Column(name = "StreetAddress", nullable = false, length = 255)
+    @Column(name = "STREET_ADDRESS", nullable = false, length = 255)
     public String getStreetAddress() {
         return streetAddress;
     }
@@ -45,7 +45,7 @@
     }
 
     @Basic
-    @Column(name = "PostIndex", nullable = false, length = 30)
+    @Column(name = "POST_INDEX", nullable = false, length = 30)
     public String getPostIndex() {
         return postIndex;
     }
@@ -78,7 +78,7 @@
     }
 
     @Basic
-    @Column(name = "UserId")
+    @Column(name = "USER_ID")
     public Long getUserId() {
         return userId;
     }
@@ -88,7 +88,7 @@
     }
 
     @Basic
-    @Column(name = "Country")
+    @Column(name = "COUNTRY")
     public String getCountry() {
         return country;
     }
@@ -98,7 +98,7 @@
     }
 
     @Basic
-    @Column(name = "Region")
+    @Column(name = "REGION")
     public String getRegion() {
         return region;
     }
@@ -108,7 +108,7 @@
     }
 
     @Basic
-    @Column(name = "City")
+    @Column(name = "CITY")
     public String getCity() {
         return city;
     }
Index: src/main/java/com/jvschool/entities/ProductCategoryEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/ProductCategoryEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/ProductCategoryEntity.java	(revision )
@@ -9,14 +9,15 @@
  * Created by Людмила on 22.07.2017.
  */
 @Entity
-@Table(name = "ProductCategory", schema = "myshop_schema")
+@Table(name = "CATEGORY", schema = "MYSHOP_SCHEMA")
 public class ProductCategoryEntity  implements Serializable {
+
     private int categoryId;
     private String categoryName;
 //    private List<ProductEntity> productsByCategoryId;
 
     @Id
-    @Column(name = "CategoryId")
+    @Column(name = "CATEGORY_ID")
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public int getCategoryId() {
         return categoryId;
@@ -27,7 +28,7 @@
     }
 
     @Basic
-    @Column(name = "CategoryName", nullable = true, length = 50)
+    @Column(name = "NAME", length = 50)
     public String getCategoryName() {
         return categoryName;
     }
Index: src/main/java/com/jvschool/entities/ProductPropertyEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/ProductPropertyEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/ProductPropertyEntity.java	(revision )
@@ -8,7 +8,7 @@
  * Created by Людмила on 22.07.2017.
  */
 @Entity
-@Table(name = "ProductProperty", schema = "myshop_schema")
+@Table(name = "PROPERTY", schema = "MYSHOP_SCHEMA")
 public class ProductPropertyEntity  implements Serializable {
     private int prodPropId;
     private String prodPropName;
@@ -16,7 +16,7 @@
     //private Set<ProductEntity> product = new HashSet<>();// = new ArrayList<>();
 
     @Id
-    @Column(name = "ProdPropId", nullable = false)
+    @Column(name = "PROPERTY_GROUP_ID", nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public int getProdPropId() {
         return prodPropId;
@@ -27,7 +27,7 @@
     }
 
     @Basic
-    @Column(name = "ProdPropName", nullable = true, length = 255)
+    @Column(name = "GROUP_NAME", nullable = true, length = 255)
     public String getProdPropName() {
         return prodPropName;
     }
@@ -60,7 +60,7 @@
     }
 
     @ManyToOne(fetch = FetchType.EAGER)
-    @JoinColumn(name = "ProdPropCatId", referencedColumnName = "PropCatId")
+    @JoinColumn(name = "PROPERTY_GROUP_ID")
     public PropertyCategoryEntity getPropertyCategoryByProdPropCatId() {
         return propertyCategoryByProdPropCatId;
     }
Index: src/main/java/com/jvschool/entities/UserEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/UserEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/UserEntity.java	(revision )
@@ -11,7 +11,7 @@
  * Created by Людмила on 21.07.2017.
  */
 @Entity
-@Table(name = "User", schema = "myshop_schema")
+@Table(name = "USER", schema = "MYSHOP_SCHEMA")
 public class UserEntity  implements Serializable {
     private long id;
     private String login;
@@ -29,7 +29,7 @@
 
 
     @Id
-    @Column(name = "Id", nullable = false)
+    @Column(name = "USER_ID", nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public long getId() {
         return id;
@@ -40,7 +40,7 @@
     }
 
     @Basic
-    @Column(name = "login", nullable = false, length = 30)
+    @Column(name = "LOGIN", nullable = false, length = 30)
     public String getLogin() {
         return login;
     }
@@ -50,7 +50,7 @@
     }
 
     @Basic
-    @Column(name = "pass", nullable = false, length = 30)
+    @Column(name = "PASSWORD", nullable = false, length = 30)
     public String getPass() {
         return pass;
     }
@@ -60,7 +60,7 @@
     }
 
     @Basic
-    @Column(name = "email", length = 30, nullable = true)
+    @Column(name = "EMAIL", length = 30)
     public String getEmail() {
         return email;
     }
@@ -70,7 +70,7 @@
     }
 
     @Basic
-    @Column(name = "FirstName", length = 30, nullable = true)
+    @Column(name = "FIRST_NAME", length = 30)
     public String getFirstName() {
         return firstName;
     }
@@ -80,7 +80,7 @@
     }
 
     @Basic
-    @Column(name = "SecondName", length = 30, nullable = true)
+    @Column(name = "LAST_NAME", length = 30)
     public String getSecondName() {
         return secondName;
     }
@@ -90,7 +90,7 @@
     }
 
     @Basic
-    @Column(name = "Birthday", nullable = true)
+    @Column(name = "BIRTHDAY")
     public Date getBirthday() {
         return birthday;
     }
@@ -130,7 +130,7 @@
     }
 
     @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)//mappedBy = "userByUserId",
-    @JoinColumn(name = "UserId", referencedColumnName = "Id")
+    @JoinColumn(name = "USER_ID")
     public List<AddressEntity> getAddresses() {
         return addresses;
     }
@@ -140,7 +140,7 @@
     }
 
     @ManyToOne(fetch = FetchType.EAGER)
-    @JoinColumn(name = "Role", referencedColumnName = "IdRole")
+    @JoinColumn(name = "ROLE_ID")
     public RoleEntity getRoleByRole() {
         return roleByRole;
     }
Index: src/main/java/com/jvschool/entities/PropertyCategoryEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/PropertyCategoryEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/PropertyCategoryEntity.java	(revision )
@@ -8,13 +8,13 @@
  * Created by Людмила on 22.07.2017.
  */
 @Entity
-@Table(name = "PropertyCategory", schema = "myshop_schema")
+@Table(name = "PROPERTY_GROUP", schema = "MYSHOP_SCHEMA")
 public class PropertyCategoryEntity  implements Serializable {
     private int propCatId;
     private String propCatName;
 
     @Id
-    @Column(name = "PropCatId", nullable = false)
+    @Column(name = "PROPERTY_GROUP_ID", nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public int getPropCatId() {
         return propCatId;
@@ -25,7 +25,7 @@
     }
 
     @Basic
-    @Column(name = "PropCatName", nullable = true, length = 255)
+    @Column(name = "NAME", nullable = true, length = 255)
     public String getPropCatName() {
         return propCatName;
     }
Index: src/main/java/com/jvschool/entities/ProductRadioPropertyEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/ProductRadioPropertyEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/ProductRadioPropertyEntity.java	(revision )
@@ -7,7 +7,7 @@
  * Created by Людмила on 23.07.2017.
  */
 @Entity
-@Table(name = "ProductRadioProperty", schema = "myshop_schema", catalog = "")
+@Table(name = "ProductRadioProperty", schema = "MYSHOP_SCHEMA", catalog = "")
 public class ProductRadioPropertyEntity {
     private int id;
     private String name;
Index: src/main/java/com/jvschool/entities/ProductEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/ProductEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/ProductEntity.java	(revision )
@@ -10,7 +10,7 @@
  * Created by Людмила on 22.07.2017.
  */
 @Entity
-@Table(name = "Product", schema = "myshop_schema")
+@Table(name = "PRODUCT", schema = "MYSHOP_SCHEMA")
 public class ProductEntity  implements Serializable {
     private long productId;
     private String productName;
@@ -34,7 +34,7 @@
 
 
     @Id
-    @Column(name = "ProductId")
+    @Column(name = "PRODUCT_ID")
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public long getProductId() {
         return productId;
@@ -45,7 +45,7 @@
     }
 
     @Basic
-    @Column(name = "ProductName", nullable = false, length = 255)
+    @Column(name = "NAME", nullable = false, length = 255)
     public String getProductName() {
         return productName;
     }
@@ -55,7 +55,7 @@
     }
 
     @Basic
-    @Column(name = "Count", nullable = false)
+    @Column(name = "COUNT", nullable = false)
     public int getCount() {
         return count;
     }
@@ -65,7 +65,7 @@
     }
 
     @Basic
-    @Column(name = "Cost", nullable = false)
+    @Column(name = "COST", nullable = false)
     public double getCost() {
         return cost;
     }
@@ -75,7 +75,7 @@
     }
 
     @Basic
-    @Column(name = "Size")
+    @Column(name = "SIZE")
     public String getSize() {
         return size;
     }
@@ -85,7 +85,7 @@
     }
 
     @Basic
-    @Column(name = "Battery")
+    @Column(name = "BATTERY")
     public String getBattery() {
         return battery;
     }
@@ -95,7 +95,7 @@
     }
 
     @Basic
-    @Column(name = "FlyTime")
+    @Column(name = "FLY_TIME")
     public String getFlyTime() {
         return flyTime;
     }
@@ -105,7 +105,7 @@
     }
 
     @Basic
-    @Column(name = "Distance")
+    @Column(name = "DISTANCE")
     public String getDistance() {
         return distance;
     }
@@ -160,7 +160,7 @@
 //    }
 
     @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
-    @JoinColumn(name = "ProductId", referencedColumnName = "ProductId")
+    @JoinColumn(name = "PRODUCT_ID", referencedColumnName = "PRODUCT_ID")
     public List<PicturesEntity> getPicturesByProductId() {
         return picturesByProductId;
     }
@@ -170,7 +170,7 @@
     }
 
     @ManyToOne(fetch = FetchType.EAGER)
-    @JoinColumn(name = "Category", referencedColumnName = "CategoryId", nullable = false)
+    @JoinColumn(name = "CATEGORY_ID", referencedColumnName = "CATEGORY_ID", nullable = false)
     public ProductCategoryEntity getProductCategoryByCategory() {
         return productCategoryByCategory;
     }
@@ -185,9 +185,9 @@
 //            joinColumns = @JoinColumn(name = "ProductId"),
 //            inverseJoinColumns = @JoinColumn(name = "ProductPropertyId"))
     @ManyToMany(cascade = CascadeType.ALL)
-    @JoinTable(name = "Prod_Prop",
-            joinColumns = @JoinColumn(name = "ProductId", referencedColumnName = "ProductId"),
-            inverseJoinColumns = @JoinColumn(name = "ProductPropertyId", referencedColumnName = "ProdPropId"))
+    @JoinTable(name = "PRODUCT_HAS_PROPERTY",
+            joinColumns = @JoinColumn(name = "PRODUCT_ID"),
+            inverseJoinColumns = @JoinColumn(name = "PROPERTY_ID"))
     public Set<ProductPropertyEntity> getProperties() {
         return properties;
     }
Index: src/main/java/com/jvschool/entities/PayWayEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/PayWayEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/PayWayEntity.java	(revision )
@@ -7,13 +7,13 @@
  * Created by Людмила on 19.07.2017.
  */
 @Entity
-@Table(name = "PayWay", schema = "myshop_schema")
+@Table(name = "PAY_WAY", schema = "MYSHOP_SCHEMA")
 public class PayWayEntity  implements Serializable {
     private int payWayId;
     private String payWayName;
 
     @Id
-    @Column(name = "PayWayId", nullable = false)
+    @Column(name = "PAY_WAY_ID", nullable = false)
     @GeneratedValue(strategy=GenerationType.AUTO)
     public int getPayWayId() {
         return payWayId;
@@ -24,7 +24,7 @@
     }
 
     @Basic
-    @Column(name = "PayWayName", nullable = false, length = 50)
+    @Column(name = "NAME", nullable = false, length = 50)
     public String getPayWayName() {
         return payWayName;
     }
Index: src/main/java/com/jvschool/entities/PicturesEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/PicturesEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/PicturesEntity.java	(revision )
@@ -6,7 +6,7 @@
  * Created by Людмила on 22.07.2017.
  */
 @Entity
-@Table(name = "Pictures", schema = "myshop_schema")
+@Table(name = "PICTURE", schema = "MYSHOP_SCHEMA")
 public class PicturesEntity {
     private long pictureId;
     private String picName;
@@ -14,7 +14,7 @@
     //private ProductEntity productByProductId;
 
     @Id
-    @Column(name = "PictureId", nullable = false)
+    @Column(name = "PICTURE_ID", nullable = false)
     //@GeneratedValue(strategy=GenerationType.AUTO)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public long getPictureId() {
@@ -26,7 +26,7 @@
     }
 
     @Basic
-    @Column(name = "PicName", nullable = true, length = 255)
+    @Column(name = "NAME")
     public String getPicName() {
         return picName;
     }
@@ -36,7 +36,7 @@
     }
 
     @Basic
-    @Column(name = "ProductId", nullable = true)
+    @Column(name = "PRODUCT_ID")
     public Long getProductId() {
         return productId;
     }
Index: src/main/java/com/jvschool/entities/PropertyRadioCategoryEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/PropertyRadioCategoryEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/PropertyRadioCategoryEntity.java	(revision )
@@ -7,7 +7,7 @@
  * Created by Людмила on 23.07.2017.
  */
 @Entity
-@Table(name = "PropertyRadioCategory", schema = "myshop_schema")
+@Table(name = "PropertyRadioCategory", schema = "MYSHOP_SCHEMA")
 public class PropertyRadioCategoryEntity {
     private int id;
     private String name;
Index: src/main/java/com/jvschool/entities/DeliveryStatusEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/DeliveryStatusEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/DeliveryStatusEntity.java	(revision )
@@ -7,13 +7,13 @@
  * Created by Людмила on 19.07.2017.
  */
 @Entity
-@Table(name = "DeliveryStatus", schema = "myshop_schema")
+@Table(name = "DELIVERY_STATUS", schema = "MYSHOP_SCHEMA")
 public class DeliveryStatusEntity implements Serializable {
     private int deliveryStatusId;
     private String deliveryStatusName;
 
     @Id
-    @Column(name = "DeliveryStatusId", nullable = false)
+    @Column(name = "DELIVERY_STATUS_ID", nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public int getDeliveryStatusId() {
         return deliveryStatusId;
@@ -24,7 +24,7 @@
     }
 
     @Basic
-    @Column(name = "DeliveryStatusName", nullable = false, length = 30)
+    @Column(name = "NAME", nullable = false, length = 30)
     public String getDeliveryStatusName() {
         return deliveryStatusName;
     }
Index: src/main/java/com/jvschool/entities/OrderEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/OrderEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/OrderEntity.java	(revision )
@@ -9,7 +9,7 @@
  * Created by Людмила on 22.07.2017.
  */
 @Entity
-@Table(name = "Order", schema = "myshop_schema")
+@Table(name = "ORDER", schema = "MYSHOP_SCHEMA")
 public class OrderEntity {
     private long orderId;
     private Timestamp dateTimeOrder;
@@ -22,7 +22,7 @@
     private List<ProductEntity> products = new ArrayList<>();
 
     @Id
-    @Column(name = "OrderId", nullable = false)
+    @Column(name = "ORDER_ID", nullable = false)
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     public long getOrderId() {
         return orderId;
@@ -34,7 +34,7 @@
 
 
     @Basic
-    @Column(name = "DateTimeOrder", nullable = false)
+    @Column(name = "TIME", nullable = false)
     public Timestamp getDateTimeOrder() {
         return dateTimeOrder;
     }
@@ -66,7 +66,7 @@
 
 
     @ManyToOne
-    @JoinColumn(name = "UserId", referencedColumnName = "Id", nullable = false)
+    @JoinColumn(name = "USER_ID", referencedColumnName = "USER_ID", nullable = false)
     public UserEntity getUser() {
         return user;
     }
@@ -76,7 +76,7 @@
     }
 
     @ManyToOne
-    @JoinColumn(name = "UserAddress", referencedColumnName = "AddressId", nullable = false)
+    @JoinColumn(name = "USER_ADDRESS_ID", referencedColumnName = "ADDRESS_ID", nullable = false)
     public AddressEntity getAddress() {
         return address;
     }
@@ -86,7 +86,7 @@
     }
 
     @ManyToOne
-    @JoinColumn(name = "PayWay", referencedColumnName = "PayWayId", nullable = false)
+    @JoinColumn(name = "PAY_WAY_ID", referencedColumnName = "PAY_WAY_ID", nullable = false)
     public PayWayEntity getPayWay() {
         return payWay;
     }
@@ -96,7 +96,7 @@
     }
 
     @ManyToOne
-    @JoinColumn(name = "DeliveryStatus", referencedColumnName = "DeliveryStatusId", nullable = false)
+    @JoinColumn(name = "DELIVERY_STATUS_ID", referencedColumnName = "DELIVERY_STATUS_ID", nullable = false)
     public DeliveryStatusEntity getDeliveryStatus() {
         return deliveryStatus;
     }
@@ -106,7 +106,7 @@
     }
 
     @ManyToOne
-    @JoinColumn(name = "DeliveryWay", referencedColumnName = "DeliveryWayId", nullable = false)
+    @JoinColumn(name = "DELIVERY_WAY_ID", referencedColumnName = "DELIVERY_WAY_ID", nullable = false)
     public DeliveryWayEntity getDeliveryWay() {
         return deliveryWay;
     }
@@ -115,10 +115,10 @@
         this.deliveryWay = deliveryStatusByDeliveryStatus;
     }
 
-    @ManyToMany(cascade = CascadeType.ALL)
-    @JoinTable(name = "Order_Product",
-    joinColumns = @JoinColumn(name = "OrderId", referencedColumnName = "OrderId"),
-    inverseJoinColumns = @JoinColumn(name = "ProductId", referencedColumnName = "ProductId"))
+    @ManyToMany
+    @JoinTable(name = "ORDER_HAS_PRODUCT",
+    joinColumns = @JoinColumn(name = "ORDER_ID"),
+    inverseJoinColumns = @JoinColumn(name = "PRODUCT_ID"))
     public List<ProductEntity> getProducts() {
         return products;
     }
Index: src/main/java/com/jvschool/entities/RoleEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jvschool/entities/RoleEntity.java	(revision dbad163c59968159794bc55e8121cb53f40ab86d)
+++ src/main/java/com/jvschool/entities/RoleEntity.java	(revision )
@@ -7,13 +7,13 @@
  * Created by Людмила on 19.07.2017.
  */
 @Entity
-@Table(name = "Role", schema = "myshop_schema")
+@Table(name = "ROLE", schema = "MYSHOP_SCHEMA")
 public class RoleEntity  implements Serializable {
     private long idRole;
     private String name;
 
     @Id
-    @Column(name = "IdRole", nullable = false)
+    @Column(name = "ROLE_ID", nullable = false)
     public long getIdRole() {
         return idRole;
     }
@@ -23,7 +23,7 @@
     }
 
     @Basic
-    @Column(name = "name", nullable = true, length = 20)
+    @Column(name = "NAME", length = 20)
     public String getName() {
         return name;
     }
